package/testing/Scripts/diff-remove-timestamps

```diff
@@ -3,11 +3,11 @@
 # Replace anything which looks like timestamps with XXXs (including the #start/end markers in logs).
 
 # Get us "modern" regexps with sed.
-if [ `uname` == "Linux" ]; then
-   sed="sed -r"
+if [ $(uname) == "Linux" ]; then
```

Comment:
Here and elsewhere: I'm fine with cleaning up these scripts, but let me point out that we tend to have various versions of these scripts sprinkled across repositories: they are often copied from some existing repo into a new one. If we start cleaning them up like this, these versions will get out of sync, which makes it harder to compare and maintain them. I'm ok with that, just wanted to mention it.

Reply:
Exact same thoughts here. I think we should remove that script altogether (in a separate PR, possibly combined with making the generated tests a feature) since it will normally be available at `$ZEEKROOT/share/btest/scripts/diff-remove-timestamps`.

Reply:
At least when installing via Homebrew on macos, this script seems to come with the bundled btest,

```console
$ fd diff-remove-timestamp /usr/local/Cellar/zeek
/usr/local/Cellar/zeek/4.2.2/share/btest/scripts/diff-remove-timestamps
```

Even though I use btest installed with `pipx` that installation doesn't seem to contain that file.

I am not sure we have a good, reliable way to get that script's location.

Reply:
Sigh, interesting, thank you. Johanna is also putting those scripts in the binary btest package. I think that's wrong since those canonifiers aren't really part of btest, they just happen to be useful for Zeek's tests. If they came with the btest Python package, they'd have to be installed elsewhere altogether.

So ... yeah. Probably best to leave that file in there.